actually got handled with https://github.com/Homebrew/homebrew-core/pull/277981 and also seeing the issue in 

https://github.com/Homebrew/homebrew-core/actions/runs/24500958594/job/71608893169?pr=277853#step:4:2856

```
  gmake[2]: *** No rule to make target '/home/linuxbrew/.linuxbrew/lib/libQt6OpenGL.so.6.10.2', needed by 'visp-check'.  Stop.
```

- #277853

--- a/Formula/o/opencv.rb
+++ b/Formula/o/opencv.rb
@@ -25,12 +25,12 @@ class Opencv < Formula
   end
 
   bottle do
-    sha256 arm64_tahoe:   "6da47b1eace0e3a0ab815a21d582cc3e4dadde2db9618407c514bc02365971ea"
-    sha256 arm64_sequoia: "8805c2677f819a3305dc3ac529a20c4de09d9c5c14ec64cde51ddde94b88274d"
-    sha256 arm64_sonoma:  "51675bfb8aeb55fe1caab6876ee2f5ae1c7ef9621be0af88e72c6cb6d2e8c09d"
-    sha256 sonoma:        "cb0d11e34c7cb0c908fe3fb97bf4426f983122fff2637f4c65f86b54629ac6c3"
-    sha256 arm64_linux:   "1314d585a6b435c53d9d1aa42c85841be78cc00059b836bbfa9eb2e40a4bf6bf"
-    sha256 x86_64_linux:  "2fe148225dc42a0512c7f451357adc906e04d7661d3504404c222ea2395a583f"
+    sha256 arm64_tahoe:   "f5ca20940550ae0583e07401f40caefc939f8fe64b279921b48ddc4cb9acf9de"
+    sha256 arm64_sequoia: "5c1a25dd27e429a8281909af699cc79e6ccf1c37b996cd18f3a057417c3f7bad"
+    sha256 arm64_sonoma:  "da1c083a9b1fe8370813fc501f585026bcde61f1579068c0b8259fc8a1a6bdb6"
+    sha256 sonoma:        "3296399eb781d8d753d418360eb125f5c655e7282d257b1810c4e8b9e826b4fa"
+    sha256 arm64_linux:   "0c834d72a0c83f34b8e0f642708304a309c4ab6e30bf544f56dcb6c39945ab19"
+    sha256 x86_64_linux:  "a148fbaa547d84070ef8537c5da2d55792b472e13d40d13bd4ff7f20babafb25"
   end
 
   head do

--- a/Formula/o/openvino.rb
+++ b/Formula/o/openvino.rb
@@ -15,12 +15,12 @@ class Openvino < Formula
   end
 
   bottle do
-    sha256 cellar: :any, arm64_tahoe:   "fcf19746e73dd4f12b9d2e0be10c9dbe1b9c721b6a25e2a8247457e59fd38f0c"
-    sha256 cellar: :any, arm64_sequoia: "b815c11d94de7ebbe8f2b5d92c7ab0e90407c5a94b9855482d344a7c445faa67"
-    sha256 cellar: :any, arm64_sonoma:  "7de80711a2fba62b8f7bdbda7a6c13f5c3baa2293dc20e6808da9d914a74f30c"
-    sha256 cellar: :any, sonoma:        "1683d253a9d0e55ff0a2f8cbfbaa85679cd05815e4cef2e99571e9068290aeb8"
-    sha256               arm64_linux:   "025771c19094a3d3148022a1283115710e32ec2ca4a07e6c97cbf4ec74c2d91c"
-    sha256               x86_64_linux:  "c32fe762988f3e8edd2da91b2ef21ebf05f5635423bd951b46d2baf007393826"
+    sha256 cellar: :any, arm64_tahoe:   "5f74efc0803c3346542baf83ab84f044f7300b2c03ce6af38c41368f9137c50f"
+    sha256 cellar: :any, arm64_sequoia: "24b49782b71130cb7a660584dbdbab8fe4a7b9ff795a6491967863ba7315b3f9"
+    sha256 cellar: :any, arm64_sonoma:  "25980f33dc5faeb376bd170f4bb0a3a1a33316f6aea72627fd236b483b6bb118"
+    sha256 cellar: :any, sonoma:        "202680f57e114e60048f32c711b5e58db7ce00655514a7bcc88665ec4e8f7e35"
+    sha256               arm64_linux:   "c5516b1e90ad77a91831ce2fbc916fac8554e690ee4a1f609fff8358f759c7a0"
+    sha256               x86_64_linux:  "d57e06366e4515ac159816bfeb8b0fc8e77391633143e31f1b66607b9df5af16"
   end
 
   depends_on "cmake" => [:build, :test]

